test/powershell/Modules/Microsoft.PowerShell.Security/ConstrainedLanguageRestriction.Tests.ps1

```diff
@@ -0,0 +1,917 @@
+            {
+                Environment.SetEnvironmentVariable("__PSLockdownPolicy", "0x80000007", EnvironmentVariableTarget.Machine);
+            }
+
+            if (revertLockdownMode)
+            {
+                Environment.SetEnvironmentVariable("__PSLockdownPolicy", null, EnvironmentVariableTarget.Machine);
+            }            
+        }
+    }
+'@
+
+    if (-not (Get-Command Invoke-LanguageModeTestingSupportCmdlet -ea Ignore))
+    {
+        $moduleName = Get-RandomFileName
+        $moduleDirectory = join-path $TestDrive\Modules $moduleName
+        if (-not (Test-Path $moduleDirectory))
+        {
+            $null = New-Item -ItemType Directory $moduleDirectory -Force
+        }
+
+        try 
+        { 
+            Add-Type -TypeDefinition $code -OutputAssembly $moduleDirectory\TestCmdletForConstrainedLanguage.dll -ErrorAction Ignore 
+        } catch {}
+
+        Import-Module -Name $moduleDirectory\TestCmdletForConstrainedLanguage.dll
+    }
+} # end if ($IsWindows)
+
+try
+{
+    $defaultParamValues = $PSDefaultParameterValues
+    $PSDefaultParameterValues["it:Skip"] = !$IsWindows
+
+    Describe "Built-ins work within constrained language" -Tags 'Feature','RequireAdminOnWindows' {
+
+        BeforeAll {
+            $TestCasesBuiltIn = @(
+                @{testName = "Verify built-in function"; scriptblock = { Get-Verb } }
+                @{testName = "Verify built-in error variable"; scriptblock = { Write-Error SomeError -ErrorVariable ErrorOutput -ErrorAction SilentlyContinue; $ErrorOutput} }
+            )
+        }
+
+        It "<testName>" -TestCases $TestCasesBuiltIn {
+
+            param ($scriptblock)
+
+            try
+            {
+                Invoke-LanguageModeTestingSupportCmdlet -EnableConstrainedLanguageMode
+
+                $result = (& $scriptblock)
+            }
+            finally
+            {
+                Invoke-LanguageModeTestingSupportCmdlet -EnableFullLanguageMode
+            }
+
+            $result.Count | Should BeGreaterThan 0
+        }
+    }
+
+    Describe "Background jobs" -Tags 'Feature','RequireAdminOnWindows' {
+
+        Context "Background jobs in system lock down mode" {
+
+            # TODO
+            # Test is pending because PowerShell Core currently stubs out Windows system lockdown APIs
+            It "Verifies that background jobs in system lockdown mode run in constrained language" -Pending {
+
+                try
+                {
+                    Invoke-LanguageModeTestingSupportCmdlet -SetLockdownMode
+
+                    $job = Start-Job -ScriptBlock { [object]::Equals("A", "B") } | Wait-Job
+                    $expectedError = $job.ChildJobs[0].Error
+                    $job | Remove-Job
+                }
+                finally
+                {
+                    Invoke-LanguageModeTestingSupportCmdlet -RevertLockdownMode
+                }
+
+                $expectedError | Should Not Be $null

```

Comment:
This check is not needed. The check below is sufficient,

Reply:
Removed.